provision-enterprise-access.sh

```diff
@@ -30,6 +30,11 @@ docker exec -t enterprise_access.app bash -c "echo 'from django.contrib.auth imp
 echo -e "${GREEN}Provisioning ${name}_worker in LMS...${NC}"
 docker exec -t edx.devstack.lms  bash -c "source /edx/app/edxapp/edxapp_env && python /edx/app/edxapp/edx-platform/manage.py lms --settings=devstack_docker manage_user ${name}_worker ${name}_worker@example.com --staff --superuser"
 
+# Create system wide enterprise role assignment
+# TODO: this is a pretty complex oneline, we should probably eventually convert this to a management command.
```

Comment:
I believe this will also be relevant to other Enterprise IDAs as well? I think you have to manually create this system-wide role assignment for every Enterprise IDA, I think. E.g., `enterprise-catalog` doesn't have this created during provisioning either.

Of course, probably not something to prioritize right now, but worth pursuing at some point.

Reply:
indeed, if we like this particular oneline, we could repeat it for enterprise-subsidy, enterprise-catalog, license-manager, and more?